docs/tests/frankenphp.bats

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bats
+
+setup() {
+  PROJNAME=my-frankenphp-site
+  load 'common-setup'
+  _common_setup
+}
+
+# executed after each test
+#teardown() {
+#  _common_teardown
+#}
+
+@test "FrankenPHP Drupal 11 quickstart with $(ddev --version)" {
+  FRANKENPHP_SITENAME=${PROJNAME}
+  run mkdir ${FRANKENPHP_SITENAME} && cd ${FRANKENPHP_SITENAME}
+  assert_success
+
+  run ddev config --project-type=drupal11 --webserver-type=generic --docroot=web --php-version=8.4
+  assert_success
+  ddev start -y
+  assert_success
+
+  cat <<'EOF' > .ddev/config.frankenphp.yaml
+web_extra_daemons:
+    - name: "frankenphp"
+      command: "frankenphp php-server --listen 0.0.0.0:80 --root=\"/var/www/html/${DDEV_DOCROOT:-}\" -v -a"
```

Comment:
```suggestion
      command: "frankenphp php-server --listen=0.0.0.0:80 --root=${DDEV_DOCROOT:-} -v -a"
```

Reply:
I'm interested in trying this out, but I think I liked my version better. This particular instance is in the test, which is less important than the quickstart

Reply:
If explicit `/var/www/html` is better then I propose this bash syntax for empty value:

```suggestion
      command: "frankenphp php-server --listen=0.0.0.0:80 --root=\"${DDEV_DOCROOT:-/var/www/html}\" -v -a"
```

Reply:
I just realized that all my suggestions here for `--root` will not use unified path, so it will be `/var/www/html` for empty value and `web` for `web` (not `/var/www/html/web`)

Let's just unify the syntax of long flags:

```suggestion
      command: "frankenphp php-server --listen=0.0.0.0:80 --root=\"/var/www/html/${DDEV_DOCROOT:-}\" -v -a"
```